pkg/controllers/interruption/controller.go

```diff
@@ -84,13 +79,18 @@ func NewController(kubeClient client.Client, clk clock.Clock, recorder events.Re
 		sqsProvider:               sqsProvider,
 		unavailableOfferingsCache: unavailableOfferingsCache,
 		parser:                    NewEventParser(DefaultParsers...),
+		cm:                        pretty.NewChangeMonitor(),
 	}
 }
 
 func (c *Controller) Reconcile(ctx context.Context, _ reconcile.Request) (reconcile.Result, error) {
 	if settings.FromContext(ctx).InterruptionQueueName == "" {
 		return reconcile.Result{RequeueAfter: time.Second * 10}, nil
 	}
+	ctx = logging.WithLogger(ctx, logging.FromContext(ctx).With("queue", settings.FromContext(ctx).InterruptionQueueName))
```

Comment:
Do think it might give more context to the user to say "Interruption Queue"?

Reply:
The logger is already named `controller.interruption` so the context of the logger already captures that this is an interruption queue